_opensearch/common-parameters.md

```diff
@@ -8,10 +8,81 @@ nav_order: 93
 
 OpenSearch supports the following parameters for all REST operations:
 
-Option | Description | Example
-:--- | :--- | :---
-Human-readable output | To convert output units to human-readable values (for example, `1h` for 1 hour and `1kb` for 1,024 bytes), add `?human=true` to the request URL. | `GET <index_name>/_search?human=true`
-Pretty result | To get back JSON responses in a readable format, add `?pretty=true` to the request URL. | `GET <index_name>/_search?pretty=true`
-Content type | To specify the type of content in the request body, use the `Content-Type` key name in the request header. Most operations support JSON, YAML, and CBOR formats. | `POST _scripts/<template_name> -H 'Content-Type: application/json`
-Request body in query string | If the client library does not accept a request body for non-POST requests, use the `source` query string parameter to pass the request body. Also, specify the `source_content_type` parameter with a supported media type such as `application/json`. | `GET _search?source_content_type=application/json&source={"query":{"match_all":{}}}`
-Stack traces | To include the error stack trace in the response when an exception is raised, add `error_trace=true` to the request URL. | `GET <index_name>/_search?error_trace=true`
+## Human-readable output
+
+To convert output units to human-readable values (for example, `1h` for 1 hour and `1kb` for 1,024 bytes), add `?human=true` to the request URL.  
+
+### Sample request
+
+The following request requires response values to be in human-readable format:
+
+```json
+
+GET <index_name>/_search?human=true
+```
+
+## Pretty result
+
+To get back JSON responses in a readable format, add `?pretty=true` to the request URL.  
+
+### Sample request
+
+The following request requires the response to be displayed in pretty JSON format:
+
+```json
+
+GET <index_name>/_search?pretty=true
+```
+
+## Content type
+
+To specify the type of content in the request body, use the `Content-Type` key name in the request header. Most operations support JSON, YAML, and CBOR formats.  
+
+### Sample request
+
+The following request specifies JSON format for the request body:
+
+```json
+
+POST _scripts/<template_name> -H 'Content-Type: application/json'
```

Comment:
Hi @smsalamon , yes! good catch! This looks like an HTTP POST request, but then they've added 
one of the cURL arguments to the end of the request. 

curl -XPUT -k -H 'Content-Type: application/json'
I need to do a bit of testing and will suggest either we replace it with a proper cURL command, or remove the incorrect request. thanks